packages/world/src/systems/DropSystem.sol

```diff
@@ -1,68 +0,0 @@
-// SPDX-License-Identifier: MIT
-pragma solidity >=0.8.24;
-
-import { System } from "@latticexyz/world/src/System.sol";
-
-import { Action } from "../codegen/common.sol";
-import { ObjectType } from "../codegen/tables/ObjectType.sol";
-import { ObjectTypeMetadata } from "../codegen/tables/ObjectTypeMetadata.sol";
-import { Position } from "../codegen/tables/Position.sol";
-import { ReversePosition } from "../codegen/tables/ReversePosition.sol";
-
-import { EntityId } from "../EntityId.sol";
-import { ObjectTypeId } from "../ObjectTypeId.sol";
-import { ObjectTypes } from "../ObjectTypes.sol";
-import { getUniqueEntity } from "../Utils.sol";
-import { Vec3 } from "../Vec3.sol";
-import { getOrCreateEntityAt } from "../utils/EntityUtils.sol";
-import { transferInventoryEntity, transferInventoryNonEntity } from "../utils/InventoryUtils.sol";
-import { DropNotification, notify } from "../utils/NotifUtils.sol";
-import { TerrainLib } from "./libraries/TerrainLib.sol";
-
-// TODO: combine the tool and non-tool drop functions
-contract DropSystem is System {
```

Comment:
Why was DropSystem removed?

Reply:
I moved both drop and pickup to inventory system, we were testing both of them together anyways, but can revert

Reply:
wait nvm spoke too soon I see `InventorySystem`